standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -97,6 +97,9 @@ public class MetastoreConf {
   static final String ACID_HOUSE_KEEPER_SERVICE_CLASS =
       "org.apache.hadoop.hive.metastore.txn.AcidHouseKeeperService";
   @VisibleForTesting
+  static final String ACID_COMPACTION_HOUSE_KEEPER_SERVICE_CLASS =
```

Comment:
simply `COMPACTION_HOUSE_KEEPER_SERVICE_CLASS`

Reply:
Done

Reply:
was it committed?
would be nice if we rename to
````
ACID_HOUSEKEEPER_SERVICE_CLASS - for aic txn
COMPACTION_HOUSEKEEPER_SERVICE_CLASS - for compaction
````